modules/enabling-the-observability-stack.adoc

```diff
@@ -0,0 +1,84 @@
+[source,yaml]
+----
+# ...
+spec:
+  monitoring:
+    managementState: Managed                 # Required: Enables and manages the observability stack
+    namespace: {monitoring-default-namespace}  # Required: Namespace where monitoring components are deployed
+    alerting: {}                              # Alertmanager configuration, uses default settings if empty
+    metrics:                                  # Prometheus configuration for metrics collection
+      replicas: 1                             # Optional: Number of Prometheus instances
+      resources:                              # CPU and memory requests and limits for Prometheus pods
+        cpulimit: 500m                        # Optional: Maximum CPU allocation in millicores
+        cpurequest: 100m                      # Optional: Minimum CPU allocation in millicores
+        memorylimit: 512Mi                    # Optional: Maximum memory allocation in mebibytes
+        memoryrequest: 256Mi                  # Optional: Minimum memory allocation in mebibytes
+      storage:                                # Storage configuration for metrics data
+        size: 5Gi                             # Required: Storage size for Prometheus data
+        retention: 90d                        # Required: Retention period for metrics data in days
+      exporters: {}                           # External metrics exporters
+    traces:                                   # Tempo backend for distributed tracing
+      sampleRatio: '0.1'                      # Optional: Portion of traces to sample, expressed as a decimal
+      storage:                                # Storage configuration for trace data
+        backend: pv                           # Required: Storage backend for Tempo traces (pv, s3, or gcs)
+        retention: 2160h                      # Optional: Retention period for trace data in hours
+      exporters: {}                           # External trace exporters
+# ...
+----
+. Click *Save* to apply your changes.
+
+.Verification
+
+Verify that the observability stack components are running in the configured namespace:
+
+. In the {openshift-platform} web console, click *Workloads* → *Pods*.
+. From the project list, select *{monitoring-default-namespace}*.
+. Confirm that there are running pods for your configuration. The following pods indicate that the observability stack is active:
++
+[source,terminal]
+----
+alertmanager-data-science-monitoringstack-#      2/2   Running   0   1m
+data-science-collector-collector-#               1/1   Running   0   1m
+prometheus-data-science-monitoringstack-#        2/2   Running   0   1m
+tempo-data-science-tempomonolithic-#             1/1   Running   0   1m
+thanos-querier-data-science-thanos-querier-#     2/2   Running   0   1m
+----
+
+. Optional: Verify that metrics are collected from instrumented components by confirming that their pods include the label `monitoring.opendatahub.io/scrape='true'`.
```

Comment:
Instrumentation refers to tracing, not sure we want to mix that when we talk about metrics.
Maybe change it to `configured`/`selected` components? Also, is this step needed when `Nest step` section below discuss that in detail?